openquake/commands/plot_assets.py

```diff
@@ -75,6 +75,19 @@ def main(calc_id: int = -1, site_model=False,
             disc = numpy.unique(dstore['discarded']['lon', 'lat'])
             p.scatter(disc['lon'], disc['lat'], marker='x', color='red',
                       label='discarded', s=markersize_discarded)
+    if show_stations and 'station_data' in dstore:
```

Comment:
I wonder if `show_stations` should be always true. I.e. if we have stations in the datastaore we should plot them always.

Reply:
I added that parameter to make the function more flexible, but I am not against removing it and simply plotting stations whenever they are available.

Reply:
Let's do that, I think that would be the normal case; if we wanted more control we can modify in the future.

Reply:
Done